Automattic/wp-calypso
# Add google domain owner banner

Fixes https://github.com/Automattic/dotcom-forge/issues/3474

## Proposed Changes

* Adds google domain owner banner

## Testing Instructions

* Flag your test user as a google domain owner via sandbox: `update_user_attribute(238083941,'is_google_domain_owner',true);`
* Confirm the banner shows up in both site specific and all domains pages:
  * http://calypso.localhost:3000/domains/manage
  * http://calypso.localhost:3000/domains/manage/testingtesterson2546.blog
* Confirm the banner does not show up on both pages if your test user is missing the flag.

![Screenshot_2023-08-30_15-23-08](https://github.com/Automattic/wp-calypso/assets/811776/3e31daf2-7c83-480c-a17b-644f07d2a12e)
![Screenshot 2023-08-30 at 15-22-46 Domains ‹ Premium site 1 — WordPress com](https://github.com/Automattic/wp-calypso/assets/811776/01b2c95b-1ae7-4bb1-9ed0-0eea8d2410d5)
